src/api/heicConvert.ts

```diff
@@ -0,0 +1,13 @@
+import { ipcRenderer } from 'electron/renderer';
+import { isPlatformMac } from '../utils/preload';
+
+export async function convertHEIC(fileData: Uint8Array): Promise<Uint8Array> {
+    if (!isPlatformMac()) {
+        throw Error('native heic conversion only supported on mac');
+    }
+    const convertedFileData = await ipcRenderer.invoke(
+        'convert-heic',
```

Comment:
what's the command which is eventually invoked?

Reply:
this calls is handled by
>   ipcMain.handle('convert-heic', (_, fileData) => {
        return convertHEIC(fileData);
    });


 https://github.com/ente-io/bhari-frame/pull/83/files#diff-b49c6a18e63d8d58063c264aa42c5f179fd55a9b7a9cfb5211379ed52264eec5R102